src/main/java/edu/kis/powp/jobs2d/drivers/label/DriverNameLabelChangeManager.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers.label;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.observer.Publisher;
+
+public class DriverNameLabelChangeManager {
```

Comment:
This class is unnecessary:
1. It is not connected in any direct way with 'changing label' (incorrect name).
2. It is a duplicate of DriverManager (observer should be introduced there).